### PR TITLE
Update sample4.cs

### PR DESCRIPTION
**Pull request by sultanimbayev**
In OWIN version 4.0 (prerelease) you cannot compare string with Path directly. You must get its value first

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@Tratcher  30 second review please.

File: aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana/samples/sample4.cs [hunk 1]
```diff
@@ -6,12 +6,12 @@ public void Configuration(IAppBuilder app)
     app.Run(context =>
     {
         // New code: Throw an exception for this URI path.
-        if (context.Request.Path == "/fail")
+        if (context.Request.Path.Value == "/fail")
```
Comment (Tratcher) [inline aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana/samples/sample4.cs]: Path.Equals is preferred

**Comment by Rick-Anderson** [comment]
@sultanimbayev can you update this to use `Path.Equals` ?

**Comment by Rick-Anderson** [comment]
@sultanimbayev Nice fix. Thanks for the update.

**Pull request merged**